docs/getting-started/install/linux/index.md

```diff
@@ -12,7 +12,7 @@ Regardless of your distribution, you will need the Mono Project GPG signing key,
 Debian, Ubuntu, and derivatives
```

Comment:
the line I linked to should have the skeleton that you need if you do `s/vertical/horizontal/`.

re: main Download page: I image it'd be too cramped there with the layout we have right now.